m4_sessions/1811_inning/civic_tech.md

```diff
@@ -23,7 +36,40 @@
      # Small Group Activity
      # Closing Discussion
 
+1. (5-10min) Icebreaker and Introductions
+     What do people know about Civic Tech.
+          List of questions to get the discussion going.
+     (5 min) Small group discussion.
+2. (5-10 min) What is Civic Tech - presentation, powerPoint or video.  Examples of what's happening.
+3. () Activity on Designing for a specific issue/problem.  Topic and leading questions.
```

Comment:
Awesome. How will you divide up the groups?  Can you have some pre-selected topics and each of you can take a different group?